packages/keystone/src/lib/server/createExpressServer.ts

```diff
@@ -43,10 +50,7 @@ const addApolloServer = ({
 export const createExpressServer = async (
   config: KeystoneConfig,
   graphQLSchema: GraphQLSchema,
-  createContext: CreateContext,
-  dev: boolean,
-  projectAdminPath: string,
-  isVerbose: boolean = true
+  createContext: CreateContext
 ) => {
```

Comment:
This is publicly exported so it needs a major changeset and `createAdminUIServer` should probably be exported from the `system` entrypoint